MIDAS/src/hardware/GPSSensor.cpp

```diff
@@ -0,0 +1,27 @@
+#include "sensors.h"
+#include "pins.h"
+#include "MicroNMEA.h"
+#include "teseo_liv3f_class.h"
+#include "sensor_data.h"
+#include <Wire.h>
+
+// Replace "Wire" with a reference to the I2C object to be used by the sensor
+TeseoLIV3F teseo(&Wire, GPS_RESET, GPS_ENABLE);
```

Comment:
gps reset and enable are wired to a gpio expander, is it possible to construct an instance that doesn't use reset or enable

Reply:
I can look into this, most likely yeah but I wouldn't be too confident without testing on hardware

Reply:
looks like the enable is only used in a specific function, but the reset is used in the init function. We'll have to make sure that we pass in a useless dummy pin into enable so that we don't accidentally fuck shit up. For the reset, I can either ask if ehardware can allocate space for a reset or we can modify the driver to digitalwrite through the gpio expander instead since the reset isn't super time sensitive

Reply:
reset pin will probably not be able to get main esp pin space, so just make a comment within the init() of the gps driver so that we remember to change the digitalwrite to one for the gpio expander. I'll also make a note of it on my end

Reply:
Added a comment